src/index.jsx

```diff
@@ -2,7 +2,6 @@ import { StrictMode } from 'react';
 import { createRoot } from 'react-dom/client';
 import { Theme } from '@radix-ui/themes';
 import { App } from './App';
-import { Theme } from '@radix-ui/themes';
```

Comment:
I actually removed this and pushed it to main when I reviewed the last PR. I had seem Tim comment on it in Slack.